jpa/WFLY-15440_Persistence3.1_Hibernate6.0.adoc

```diff
@@ -0,0 +1,62 @@
+= Hibernate ORM 6.0 Jakarta Persistence 3.1
+:author:            Scott Marlow
+:email:             smarlow@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Support Persistence 3.1 by including Hibernate ORM 6.0.x https://issues.redhat.com/browse/WFLY-15440/[WFLY-15440] + https://issues.redhat.com/browse/EAP7-1815[EAP7-1815].  Also update the WildFly Persistence Container to support Jakarta Persistence 3.1.
+
+The list of databases to test with are in comments on https://issues.redhat.com/browse/EAP7-1820.
+
+Consider including Hibernate https://github.com/hibernate/hibernate-orm/releases/tag/6.1.0[ORM 6.1.0] instead of Hibernate ORM 6.0.x.
```

Comment:
Is this a requirement now, not just a consideration?

AIUI, moving to 6.1.x  is required to:

1) Resolve security manager failures: https://issues.redhat.com/browse/WFLY-16552

2) Support Jandex 3, which is both desirable in general for WF, but is also a requirement for https://issues.redhat.com/browse/WFLY-16959, the move of Smallrye Open API to a .Final version, which is a blocker.

Reply:
Answering myself; it is a requirement now.